src/Features/Core/Portable/Completion/Providers/AbstractMemberInsertingCompletionProvider.cs

```diff
@@ -188,7 +191,6 @@ private TextSpan ComputeDestinationSpan(SyntaxNode insertionRoot)
 
             var dismemberedDocument = document.WithSyntaxRoot(root);
 
-            dismemberedDocument = await Simplifier.ReduceAsync(dismemberedDocument, Simplifier.Annotation, null, cancellationToken).ConfigureAwait(false);
```

Comment:
why was this a problem here  why would reduction remove imports?

Reply:
After overriding member is removed, added usings become unnecessary.

Reply:
i see :)